### PR TITLE
updated php-soap instructions for sharepoint.

### PR DESCRIPTION
**Pull request by jnweiger**
Fix php prerequisites in the sharepoint documentation.

Backport to 10.10, 10.9 and 10.8

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/external_storage/sharepoint-integration_configuration.adoc [hunk 1]
```diff
@@ -19,8 +19,8 @@ properly accessed as per SharePoint rules.
 
 The plugin uses the Simple Object Access Protocol (SOAP) and WebDAV for
 the uploads and downloads to talk to SharePoint servers. Your ownCloud
-server must have `php-soap` or `php5-soap` installed. Linux packages and
-ownCloud appliances will install `php5-soap` as a required dependency.
+server must have `php-seclib` and `php-soap` or `php7-soap` installed; and enabled soap as a php module.
+A webserver restart may be needed after changing php modules.
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/external_storage/sharepoint-integration_configuration.adoc:23]:
```suggestion
server must have the php modules `php-seclib` and `php-soap` installed and enabled, a webserver restart may be needed after adding/changing php modules.
```

File: modules/admin_manual/pages/enterprise/external_storage/sharepoint-integration_configuration.adoc [hunk 2]
```diff
@@ -19,8 +19,8 @@ properly accessed as per SharePoint rules.
 
 The plugin uses the Simple Object Access Protocol (SOAP) and WebDAV for
 the uploads and downloads to talk to SharePoint servers. Your ownCloud
-server must have `php-soap` or `php5-soap` installed. Linux packages and
-ownCloud appliances will install `php5-soap` as a required dependency.
+server must have `php-seclib` and `php-soap` or `php7-soap` installed; and enabled soap as a php module.
```
Comment (jnweiger) [inline modules/admin_manual/pages/enterprise/external_storage/sharepoint-integration_configuration.adoc:22]: not sure about php-seclib.
The actual package name is php-phpseclib but then it still does not seem to work. 
More investigation needed.

**Comment by mmattel** [comment]
not sure about php-seclib.
The actual package name is php-phpseclib but then it still does not seem to work.
More investigation needed.

File: modules/admin_manual/pages/enterprise/external_storage/sharepoint-integration_configuration.adoc [hunk 3]
```diff
@@ -19,8 +19,8 @@ properly accessed as per SharePoint rules.
 
 The plugin uses the Simple Object Access Protocol (SOAP) and WebDAV for
 the uploads and downloads to talk to SharePoint servers. Your ownCloud
-server must have `php-soap` or `php5-soap` installed. Linux packages and
-ownCloud appliances will install `php5-soap` as a required dependency.
+server must have the php modules `php-phpseclib` and `php-soap` installed
+and `phpenmod soap` enabled. A webserver restart may be needed after adding/changing php modules.
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/external_storage/sharepoint-integration_configuration.adoc:23]:
```suggestion
and the SOAP module enabled with `phpenmod soap`. A webserver restart may be required after adding/changing php modules.
```

**Review by mmattel — APPROVED** [review]

**Pull request merged**